exercises/practice/allergies/.meta/example.nim

```diff
@@ -1,24 +1,16 @@
-import sequtils
-
 type
-  Allergies* = object
-    score*: int
-
-var
-  allergiesList = [
-    "eggs",
-    "peanuts",
-    "shellfish",
-    "strawberries",
-    "tomatoes",
-    "chocolate",
-    "pollen",
-    "cats"
-  ]
-
+  Allergen* = enum
+    Eggs
+    Peanuts
+    Shellfish
+    Strawberries
+    Tomatoes
+    Chocolate
+    Pollen
+    Cats
 
-proc isAllergicTo*(allergies: Allergies, allergy: string): bool =
-  (allergies.score and 1 shl allergiesList.find(allergy)) != 0
+func isAllergicTo*(allergen: Allergen, score: int): bool =
+  (score and 1 shl allergen.ord) != 0
```

Comment:
I think we should use `bitops.testbit` here

Reply:
I agree in the other 2 situations, but here, IMO, it's different. The point over here is to realize that you can just test the bit. I don't see it relevant in knowing how to test the bit with `and` or `shift`s. It still reveals an understanding of the fundamentals of the exercise and concepts.

Definitely non-blocking as its not user-facing code.

Reply:
I will still argue that `testBit` already encompasses the heart of the exercise. That shift is core. Either way I'm approving. I see your point